tools/jenkins/run_performance_profile_daily.sh

```diff
@@ -30,7 +30,7 @@
 
 set -ex
 
-cd $(dirname $0)/../../..
+cd $(dirname $0)/../..
 
 CPUS=`python -c 'import multiprocessing; print multiprocessing.cpu_count()'`
```

Comment:
Is `CPUS` actually used? If so, should it be added to `run_performance_profile_hourly.sh`? If not used, let's remove it.